application/server/notification/scan_notifier.go

```diff
@@ -117,6 +117,34 @@ func (n *scanNotifier) appendOssIssues(scanIssues []lsp.ScanIssue, folderPath st
 			continue // skip non-oss issues
 		}
 
+		matchingIssues := make([]lsp.OssIssueData, len(additionalData.MatchingIssues))
```

Comment:
**Non-blocking**: I noticed that `folderPath` isn't used in this function, as well as in others called from the same place.:

```go
	if pr == product.ProductInfrastructureAsCode {
		scanIssues = n.appendIacIssues(scanIssues, folderPath, issues)
	} else if pr == product.ProductCode {
		scanIssues = n.appendCodeIssues(scanIssues, folderPath, issues)
	} else if pr == product.ProductOpenSource {
		scanIssues = n.appendOssIssues(scanIssues, folderPath, issues)
	}
```

Maybe we could clean them up too? We could simplify the function signatures by removing it?

Reply:
I noticed that too but I'm already changing a lot of code and I had to set a limit to the number of refactorings I can do 😓